atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/knowledge/KnownConcludedTransactions.java

```diff
@@ -41,6 +41,14 @@ public interface KnownConcludedTransactions {
      */
     void addConcludedTimestamps(Range<Long> knownConcludedInterval);
 
+    /**
+     * Returns the greatest known concluded timestamp for which transaction is known to have concluded. This call
+     * relies on local cache. Hence, it is possible for the view to be out of date.
+     *
+     * @return the greatest known concluded timestamp for which transaction is known to have concluded.
+     */
+    long lastKnownConcludedTimestamp();
```

Comment:
nit: maybe `lastLocallyKnownConcludedTimestamp` to be explicit this may not be current?